packages/postgres-database/src/simcore_postgres_database/utils_products_prices.py

```diff
@@ -59,7 +59,7 @@ async def get_product_latest_stripe_info(
         )
     ).fetchone()
     if row is None:
-        msg = "No product Stripe info defined in database"
+        msg = f"No product Stripe info defined in database [{product_name=}]"
```

Comment:
this sentence is incorrect. I am not sure what it is supposed to mean?
that something is wrongly defined in the database for product_name= ? we are talking about some configuration?